### PR TITLE
Fix textfield layout on basic auth credentials page

### PR DESCRIPTION
**Pull request by erikjv**
Fixes: #11913

### CODE REVIEW

File: src/gui/qml/credentials/BasicAuthCredentials.qml [hunk 1]
```diff
@@ -46,6 +46,9 @@ Credentials {
         Keys.onBacktabPressed: {
             widget.parentFocusWidget.focusPrevious();
         }
+
```
Comment (TheOneRing) [inline src/gui/qml/credentials/BasicAuthCredentials.qml:49]: We are using a layout and must not mix anchors with layouts.

**Review by TheOneRing — CHANGES_REQUESTED** [review]
^

File: src/gui/qml/credentials/BasicAuthCredentials.qml (hunk 1 above)
Comment (erikjv) [reply src/gui/qml/credentials/BasicAuthCredentials.qml:51]: done

File: src/gui/qml/credentials/BasicAuthCredentials.qml [hunk 2]
```diff
@@ -30,43 +30,51 @@ Credentials {
...
-        Keys.onBacktabPressed: {
-            widget.parentFocusWidget.focusPrevious();
-        }
-    }
+        ColumnLayout {
+            Label {
+                text: credentials.userNameLabel
+            }
+            TextField {
+                id: userNameField
```
Comment (TheOneRing) [inline src/gui/qml/credentials/BasicAuthCredentials.qml:43]: I think now that we have this internal Panel, with this new layout, you should set fillWidth for the contained elements.
Also I'm pretty sure you should add the buttons, else weird stuff will happen.

Comment (erikjv) [reply src/gui/qml/credentials/BasicAuthCredentials.qml:43]: done

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**